### ISSUE TITLE
Router now requires pino-elasticsearch

### ISSUE THREAD

**Issue by cjus** [flywheelsports/hydra-router#37]
Reported by @taurenk 

```
/Users/tkristich/Projects/fwsp-hydra-router/node_modules/fwsp-logger/lib/PinoLogger.js:146
      throw new Error(`Failed to start pino-elasticsearch: ${err}`);
      ^

Error: Failed to start pino-elasticsearch: Error: spawn pino-elasticsearch ENOENT
```

Temporary fix is to install pino-elasticsearch via:

```shell
$ npm install -g pino-elasticsearch
```

Need to patch code or add to package.json for a proper fix.

**Comment by cjus** [comment]
@emadum please review and determine if this is still an issue on Windows PCs.

**Comment by emadum** [comment]
This is still a problem. I'm going to push a fix to fwsp-logger for this right now.

**Comment by emadum** [comment]
Fixed in fwsp-logger 0.1.5.

**Comment by cjus** [comment]
Cool thanks. I'll update Hydra-Router.

**Comment by cjus** [comment]
Fixed in Hydra-router 1.2.8
